AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/justice-user-form/justice-user-form.component.html

```diff
@@ -64,11 +64,31 @@ <h1 class="govuk-fieldset__heading">{{ mode === 'edit' ? 'Edit role' : 'Add a ju
       />
     </div>
     <div class="govuk-form-group">
-      <label class="govuk-label" for="role"> Role </label>
-      <select class="govuk-select" id="role" formControlName="role" name="role">
-        <option value="{{ availableRoles.Vho }}">CSO</option>
-        <option value="{{ availableRoles.VhTeamLead }}">Administrator</option>
-      </select>
+      <label class="govuk-label"> Roles </label>
+      <p *ngIf="form.controls.roles.invalid" id="roles-error" class="govuk-error-message">
+        <span class="govuk-visually-hidden">Error:</span> {{ form.controls.roles.errors.userRoleInvalid }}
+      </p>
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-one-third">
+          <input #checkBoxVho id="role_vho" name="role_vho" type="checkbox" formControlName="rolevho" (change)="onCheckBoxChange()" />
```

Comment:
The justice user form is a reactive form. Manually adding this and then an onChange method looks very hacky.

Consider using FormArray and looping through an array of available roles rather than checking whether or not a property exists and then building an array.

Reply:
done